Rpm winget private repo azure (#61)
microsoft/winget-cli-restsource: README.md

# Windows Package Manager Repository

## Create a new private repository in Azure

> To simply the creation of a Windows Package Manager private repository, the `automation.ps1` [PowerShell script](Tools/Automation.ps1) is offered. This script will create new Elements required to host a Windows Package Manager private repository in Azure.

Reviewer: ```suggestion
To simplify the creation of a Windows Package Manager private source, the `automation.ps1` [PowerShell script](Tools/Automation.ps1) is offered. This script will create new resources required to host a Windows Package Manager private source in Azure.
```
Suggest aligning with azure resource names.

Are we aligning on the name 'repository' for any collection of apps even if it is not a github repository? Or is this a source which contains apps?

Author: I confirmed with Demitrius and we are aligning with "source". I'll update the documents

The `automation.ps1` script has the following parameter inputs:
| Required | Parameter          | Description                                                                                                                |
|----------|--------------------|----------------------------------------------------------------------------------------------------------------------------|
| Yes      | ResourcePrefix     | A string of letters which will be prefixed to your newly created Azure resources.                                          |

> | Yes      | Index              | A string of letters or numbers which will be sufixed to your newly created Azure resources.                                |

Reviewer: ```suggestion
| Yes      | Index              | A string of letters or numbers which will be suffixed to your newly created Azure resources.                                |
```
typo

| Yes      | AzResourceGroup    | The Resource Group that will be used to contain the Azure resources.                                                       |

> | No       | AzSubscriptionName | The name of the Azure Subscription that will be used to pay for the Azure resources.                                       |

Reviewer: how is the subscription not required?

Author: The Subscription name is not required if the default (or only assigned) Subscription is the correct target Subscription. A Subscription is required to bill against, but does not need to be declared on script run.

Reviewer 2: Can you add that to the doc? Because I was confused by the same thing. If you can add a link to how to set the default subscription, that would be great as well.

Author: Provided a similar comment to what I described above into the script identifying why this parameter is optional. Also included a link to setting Azure preferences

| No       | AzLocation         | The Azure location where the Azure resources will be created. (Default: westus)                                            |

> | No       | WorkingDirectory   | The folder location that contains this the ARM template files, as well as where the Azure Parameter files will be created. |

Reviewer: ```suggestion
| No       | WorkingDirectory   | The folder location that contains the ARM template files, as well as where the Azure Parameter files will be created. |
```

> For more information on how to manually implement your Windows Package Manager in Azure, please visit our Docs.

Reviewer: Can we add a link to which docs?

Author: I've created a task to link this to the future documentation (See ".\docs\new-private-repository-azure.md" for reference to documentation that will be in Docs in the future.)
